package.json

```diff
@@ -47,6 +46,7 @@
   },
   "dependencies": {
     "@azure/cosmos": "^3.6.3",
+    "@azure/identity": "^4.0",
```

Comment:
did you manually edit the package.json to add this? anything installed via `npm i <package>` should end up with a three part version here